calculator/main.py

```diff
@@ -58,18 +84,21 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    return calc['history']
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history'] = []
 
 
 def repeat_last_operation(calc):
     """
     Returns the result of the last operation executed in the history.
     """
-    pass
+    history = calc['history']
+    if not history:
+        return None
+    return history[-1][-1]
```

Comment:
This one bugged me for awhile. I eventually came up with history[-1][3] but I like yours better.